.github/workflows/integration.yaml

```diff
@@ -1,74 +1,184 @@
-name: Integration Tests For Retina
+name: Build images and run E2E tests. 
```

Comment:
Since the images are built and pushed in the `Build` pipeline, why duplicate that effort? We can add steps to run it in `images.yaml` itself?

Reply:
if we run it in images.yaml, every pr will run this tests, we want it to only run when triggered by owners, but we want the build pipeline to run for every PR.